tests/wandb_artifacts_test.py

```diff
@@ -24,6 +24,7 @@ def __init__(self, name="my_object.pb", metadata=None, version_id=None):
             self.name = name
             self.key = name
             self.content_length = 10
+            self.content_type = "application/pb; charset=UTF-8"
```

Comment:
Ideally, we would have a test case for when `x-directory` is set on the ref being added.